istio.deps

```diff
@@ -12,6 +12,6 @@
 		"repoName": "proxy",
 		"prodBranch": "master",
 		"file": "",
-		"lastStableSHA": "80ec4f6c2a672b00e55d34ef9b5d26a42a98da42"
+		"lastStableSHA": "9b4cbc6559822311d9bc04a6e0dd129b9b75617b"
```

Comment:
Could you update this to 9b4cbc6559822311d9bc04a6e0dd129b9b75617b? We would like to include that PR.

Reply:
it already is that one :-)

Reply:
Thanks for correcting me.